blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpAggregatedIngestionHandler.java

```diff
@@ -99,10 +99,10 @@ public void handle(ChannelHandlerContext ctx, HttpRequest request) {
                         HttpResponseStatus.BAD_REQUEST );
             }
 
-        } catch (JsonParseException ex) {
+        } catch (JsonSyntaxException ex) {
```

Comment:
Note, for some reason, we are using Jackon for parsing json in *HttpMetricsIngestionHandler*, but Gson for parsing json in *HttpAggregatedIngestionHandler* and *HttpAggregatedMultiIngestionHandler*.

So this exception catching has been wrong because since the Gson class which throws JsonSyntaxException, not JsonParseException (thrown by codehaus.Jackson.TypeFactory).  We were never going to catch any JsonParseException here, so it's valid but a useless catch statement.

We now have better json parsing error message for aggregated metrics ingestion, and not just a generic "Failed parsing content" as the response from the catch-all exception.